app/src/sharedTest/java/org/oppia/android/app/recyclerview/BindableAdapterTest.kt

```diff
@@ -440,12 +445,17 @@ class BindableAdapterTest {
       testCoroutineDispatchers.runCurrent()
 
       // The updated live data value should be reflected on the UI due to the bound lifecycle owner.
-      onView(atPosition(R.id.test_recycler_view, 0)).check(matches(withText("new value")))
+      onView(
+        atPosition(
+          R.id.test_recycler_view,
+          position = 0
+        )
+      ).check(matches(withText("new value")))
     }
   }
 
   private fun setUpTestApplicationComponent() {
-    ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
+    ApplicationProvider.getApplicationContext<TestApplication>().inject(bindableAdapterTest = this)
```

Comment:
let's not add named argument for this.

Reply:
Removed named argument for bindableAdapterTest: https://github.com/oppia/oppia-android/pull/3153/commits/431cf43838b96dca47cfe23de07f460bd53d4ed3